.gitignore

```diff
@@ -12,4 +12,5 @@
 /.sass-cache
 /venv
 /lib/cassandra_murmur3.*
-/node_modules
\ No newline at end of file
+/node_modules
+/.idea
```

Comment:
put this into your global gitignore file instead https://help.github.com/articles/ignoring-files/#create-a-global-gitignore